### PR TITLE
chore: ELY-1051 support labels in cloud-run

### PR DESCRIPTION
**Pull request by tomascarlqvist**

### CODE REVIEW

**Review by anakovt — APPROVED** [review]

File: cloud-run/README.md [hunk 1]
```diff
@@ -274,3 +275,24 @@ The following example enables `canary` for  service. New revision of a service w
 Instead traffic percentage will be updated each `canary.interval` minutes using `canary.steps` values.
 On each step metrics for latest revision will be fetched and compared to `canary.thresholds`.
 If current metrics do not exceed thresholds, traffic will be increased otherwise servie will rollback to previous revision.
+
+#### Deployment with labels
+
+The following example shows how to add labels to the service. `component`, `iso-country`, `product` and `tenant-alias` are used in reports and must be on the format below. `iso-county` is the two lower case letters country code and should only be used in country specific services. `tenant-alias` is the alias for the tenant that can be found in iam, (plantagen, isrg) and should only be used in tenant specific services. `product` is the product the service belong to. `component` is the product component that the service belong to. It is possible to also add custom labels as seen below.
```
Comment (sasjo) [inline cloud-run/README.md:281]: Remove plantagen and isrg mentions from this public README.

File: cloud-run/README.md [hunk 2]
```diff
@@ -274,3 +275,24 @@ The following example enables `canary` for  service. New revision of a service w
...
+Given the following `cloud-run.yaml`:
+```yaml
+name: my-service
+memory: 256Mi
+cpu: 1
+labels:
+  component: checkout-api
+  iso-country: se
+  product: hii-checkout
+  tenant-alias: plantagen
```
Comment (sasjo) [inline cloud-run/README.md:292]: Use more generic examples to not expose Extenda specifics or real tenant IDs.

**Review by sasjo — CHANGES_REQUESTED** [review]
Minor changes to docs before merge.

I'm not sure how you are enforcing kebab-case on the custom labels. I think this can be expressed in the schema using property regex rules https://json-schema.org/understanding-json-schema/reference/object.html#pattern-properties

File: cloud-run/README.md (hunk 2 above)
Comment (tomascarlqvist) [reply cloud-run/README.md]: Fixed

**Comment by tomascarlqvist** [comment]
> Minor changes to docs before merge.
> 
> I'm not sure how you are enforcing kebab-case on the custom labels. I think this can be expressed in the schema using property regex rules https://json-schema.org/understanding-json-schema/reference/object.html#pattern-properties

Fixed

**Review by sasjo — APPROVED** [review]

**Pull request merged**